build.gradle.kts

```diff
@@ -95,4 +95,4 @@ fun replaceVersion(path: String) {
 }
 
 replaceVersion("README.md")
-replaceVersion("docs/pages/getting_started.mdx")
\ No newline at end of file
+replaceVersion("docs/pages/getting_started.mdx")
```

Comment:
Please undo removing the new line on these files.

Reply:
This should be additions, my IntelliJ is configured to always end files with an empty newline